tutorials/mesh/conv_meshcutoff.lua

```diff
@@ -0,0 +1,124 @@
+--[[
```

Comment:
You are mixing many things at once. It is easier for me to see what belongs where if we could have separated these things.
1 for tutorials, 1 for NEB stuff and 1 for documentation